src/systems/physics/EntityFeatureMap_TEST.cc

```diff
@@ -92,7 +92,8 @@ class EntityFeatureMapFixture: public InternalFixture<::testing::Test>
   public: EnginePtrType engine;
 };
 
-TEST_F(EntityFeatureMapFixture, AddCastRemoveEntity)
+// See https://github.com/ignitionrobotics/ign-gazebo/issues/1175
+TEST_F(EntityFeatureMapFixture, IGN_UTILS_TEST_ENABLE_ONLY_LINUX(AddCastRemoveEntity))
```

Comment:
Can this be disabled only on Windows?

Reply:
Sure, my bad there. Addressed here: 5901c2b